tests/x11/yast2_snapper.pm

```diff
@@ -72,101 +31,26 @@ sub run() {
     # Start an xterm as root
     x11_start_program("xterm");
     assert_screen "xterm";
+    # Disable screen lock and blank screen for current Gnome session
+    script_run("gsettings set org.gnome.desktop.session idle-delay 0");
```

Comment:
If the module is not triggered on kde or other desktop environments then we do not need to care. There are also other commands in some other modules to disable the screensaver and blanking. Please compare to the other occassions.